forge-gui/res/cardsfolder/upcoming/call_to_the_void.txt

```diff
@@ -0,0 +1,7 @@
+Name:Call to the Void
+ManaCost:4 B
+Types:Sorcery
+A:SP$ ChooseCard | Defined$ Player | Choices$ Creature | SecretlyChoose$ True | Amount$ 1 | ControlAndNot$ True | ChoiceTitle$ Secretly choose a creature | Reveal$ True | RevealTitle$ OVERRIDE Chosen creatures. They will be destroyed. | SubAbility$ DBDestroyChosen | Mandatory$ True
+SVar:DBDestroyChosen:DB$ DestroyAll | ValidCards$ Creature.ChosenCard | SubAbility$ DBCleanUp
+SVar:DBCleanUp:DB$ Cleanup | ClearRemembered$ True
```

Comment:
Now you no longer need the Cleanup :)

Reply:
I had actually reviewed the script a few times to try to make sure I wasn't forgetting anything this time...😅 Oh well! Thank you for the feedback!